scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/IncrementalSimulationDriver.java

```diff
@@ -79,6 +81,27 @@ private void initSimulation(){
     }
   }
 
+  //
+  private void simulateUntil(Duration endTime){
+    assert(endTime.noShorterThan(curTime));
```

Comment:
I'd avoid using `assert` since assertion checking can be toggled by the runner of the JVM.
I think it may be best to replace this with: `if (...) throw new Error("...")`.

Also, line 84 seems to have a stray empty comment.

Reply:
I'll replace the assert with an exception and remove the comment.

Reply:
Cool, thanks Adrien!